recipes/clikit/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "clikit" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec47b987494320acc0739769f5ae6e092ddba7e0e1a9e2f36c705eabd95a266b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pastel >=0.1.0,<0.2.0
+    - pip
+    - pylev >=1.3,<2.0
+    - python
+    - poetry >=0.12
+# clikit need poetry to build
+# pip install clikit==0.1.0 --no-binary :all:
+
+  run:
+    - pastel >=0.1.0,<0.2.0
+    - pylev >=1.3,<2.0
+    - python
```

Comment:
It looks like your missing dependencies on Python 2.7?
https://github.com/sdispater/clikit/blob/master/pyproject.toml#L23-L27

Reply:
I think `enum34` is also needed?

Reply:
it dont need enum34 in version 0.1.0
Enum34 will be included in next version 0.2.0